### DISCUSSION TITLE
With TextInput prefix value not stored into my database. How to send it to my database with user input?

### DISCUSSION THREAD

**Discussion by CodeWithPranta** [filamentphp/filament#8913] [General]
` Forms\Components\TextInput::make('roll')
                    ->required()
                    ->prefix(function (callable $get){
                        $class = StudentClass::find($get('student_class_id'));
                        $group = ClassGroup::find($get('class_group_id'));

                        if (!$class || !$group)
                        {
                            return 'NA';
                        }

                        // Extract the first letter of each word and concatenate them
                        $result = '';
                        $words = explode(' ', $class->class_name . ' ' . $group->group_name);
                        foreach ($words as $word) {
                            $result .= strtoupper(substr($word, 0, 1));
                        }

                        return $result;
                    })`
Here I want prefix value which comes from my class and group field dynamically will be stored into my database. Suppose Admin select class 9 and group science then prefix will be C9S + user input. But prefix not send into my database.

**Comment by leandrocfe** [comment]
https://filamentphp.com/docs/3.x/panels/resources/creating-records#customizing-data-before-saving
